dll/cpl/timedate/timedate.c

```diff
@@ -44,7 +44,7 @@ VOID DisplayWin32Error(DWORD dwErrorCode)
                    NULL );
 
 #if DBG
-    if (swprintf(szMsg, L"%hs:%d: %s", file, line, lpMsgBuf))
+    if (swprintf(szMsg, L"%hs:%d: %s", file, line, (PWSTR)lpMsgBuf))
```

Comment:
Hm why not just changing the type of lpMsgBuf to a (L)PWSTR directly instead of using LPVOID? This would avoid this cast there.

Reply:
... and would instead introduce a cast in the `FormatMessage` call? The one here is the safer cast, so this seems preferable.

Reply:
It wouldn't introduce another cast in FormatMessage().